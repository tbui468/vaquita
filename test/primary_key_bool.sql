create database sol;
open sol;

create table planets (id bool key, name string, mass int, atmosphere bool);
insert into planets (id, name, mass, atmosphere) values 
    (true, "Mars", 10, true), 
    (false, "Venus", 10, false);
select * from planets;
drop table planets;

close sol;
drop database sol;
exit;
