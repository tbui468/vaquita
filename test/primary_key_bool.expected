created database sol
opened database sol
created table planets
inserted 2 record(s) into planets
id, name, mass, atmosphere
false, Venus, 10, false
true, Mars, 10, true
dropped table planets
closed database sol
dropped database sol
